core/src/main/java/com/google/googlejavaformat/java/CommandLineOptionsParser.java

```diff
@@ -99,6 +110,25 @@ static CommandLineOptions parse(Iterable<String> options) {
     return optionsBuilder.build();
   }
 
+  private static void parseSourcesFile(ImmutableList.Builder<String> filesBuilder, String value) {
+    Path path = Paths.get(value);
+    try {
+      for (String line : Files.readAllLines(path, StandardCharsets.UTF_8)) {
+        line = line.trim();
```

Comment:
I wonder if this line would be less bug-prone if it were replaced with `CharMatcher.whitespace().trimFrom(line)`.